_security-plugin/index.md

```diff
@@ -8,20 +8,66 @@ redirect_from:
+
+Another notable characteristic of access control in OpenSearch is the ability to assign user access through levels of increasing granularity. Fine grained access control (FGAC) means that a role can control permissions for users at not only the cluster level, but the index level, the document level, and even the field level. For example, a role may provide a user access to certain cluster-level permissions but at the same time prevent the user from accessing a given group of indexes. Likewise, that role may grant access to certain types of documents but not others, or even include access to specific fields within a document but exclude access to other sensitive fields. Field masking further extends the fine grained control of data access by providing options to mask certain types of data, such as a list of emails, which can still be aggregated but not made viewable to a role.
+
+To learn more about this feature, see the [Access control]({{site.url}}{{site.baseurl}}/security-plugin/access-control/index/) section of security documentation.
+
+### Audit logging and compliance
+
+Finally, audit logging and compliance refer to mechanisms that allow for tracking and analysis of activity within the cluster. This is important after events in which sensitive data is breached (unauthorized access) or suffers unintended exposure, as when data is sent to an unsecure location and left vulnerable. However, audit logging can be just as valuable a tool for assessing excessive loads on the cluster or surveying trends for a given task. This feature allows you to oversee changes made anywhere in the cluster and track access patterns and API requests of all types, whether valid or invalid.
+
+How OpenSearch archives logging is configurable at many levels of detail, and there are a number of options for where those logs are stored. Compliance features also ensure that all data is available if and when compliance auditing is required. In this case, the logging can be automated to focus on data especially pertinent to those compliance requirements.
+
+See the [Audit logs]({{site.url}}{{site.baseurl}}/security-plugin/audit-logs/index/) section of security documentation to read more about this feature.
+
+## Other features and functionality
+
+Security includes other features that make working in OpenSearch easier.
+
+### Dashboards multi-tenancy
+
+One such feature is OpenSearch Dashboards multi-tenancy. Tenants are work spaces that include visualizations, index patterns, and other Dashboards objects, which can be shared with other Dashboards users. OpenSearch leverages roles to manage accesss to tenants and safely make them available to other Dashboards users.
```

Comment:
I think this should be "...and other Dashboards objects and can be shared...". In the third sentence, remove the extra s.

Reply:
How's this: "Tenants are work spaces that include visualizations, index patterns, and other Dashboards objects. Multi-tenancy allows for the sharing of tenants among users of Dashboards and leverages OpenSearch roles to manage access to tenants and safely make them available to others."